anesthetic/samples.py

```diff
@@ -8,6 +8,8 @@
 import pandas
 import copy
 import warnings
+from scipy.interpolate import interp1d
+from scipy.optimize import minimize_scalar
```

Comment:
Hi Stefan, going on the `CI / lint details` gives you the error message:

> anesthetic/samples.py:11:1: F401 'scipy.interpolate.interp1d' imported but unused
> anesthetic/samples.py:12:1: F401 'scipy.optimize.minimize_scalar' imported but unused

Are you using `interp1d` and `minimize_scalar` anywhere in `anesthetic/samples`? If not, these lines should be removed.

Reply:
Thanks @lukashergt !